From: Michael Davidsaver <mdavidsaver@bnl.gov>
Date: Mon, 25 Mar 2013 10:51:54 -0400
Subject: stream: remove 3.13 build

equivalent to StreamDevice-2-6-patch20121009
---
 streamApp/Makefile | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/streamApp/Makefile b/streamApp/Makefile
index 1981c42..bbb2f1c 100644
--- a/streamApp/Makefile
+++ b/streamApp/Makefile
@@ -1,13 +1,5 @@
 TOP=../..
 
-# Look if we have EPICS R3.13 or R3.14
-ifeq ($(wildcard $(TOP)/configure),)
-# EPICS R3.13
-include $(TOP)/config/CONFIG_APP
-# The real work is in Makefile.Vx
-include $(TOP)/config/RULES_ARCHS
-else
-
 # EPICS R3.14
 include $(TOP)/configure/CONFIG
 
@@ -68,13 +60,7 @@ CPPFLAGS += -DDEBUGFILE=StreamDebug.log
 
 include $(TOP)/configure/RULES
 
-ifeq ($(BASE_3_14), YES)
 clean:: myclean
-else
-clean: myclean
-endif
 
 myclean:
 	$(RM) core* StreamDebug.log
-
-endif
