From: Martin Konrad <konrad@frib.msu.edu>
Date: Fri, 15 Mar 2019 09:31:23 -0400
Subject: Customize configure/RELEASE

---
 configure/RELEASE | 13 +++++++++----
 1 file changed, 9 insertions(+), 4 deletions(-)

diff --git a/configure/RELEASE b/configure/RELEASE
index a575921..9d445c5 100644
--- a/configure/RELEASE
+++ b/configure/RELEASE
@@ -17,10 +17,15 @@ TEMPLATE_TOP=$(EPICS_BASE)/templates/makeBaseApp/top
 # For SynApps:
 SUPPORT=
 
-EPICS_BASE=/usr/local/epics/base-7.0.1
-ASYN=~/top-7/asyn4-33
-CALC=~/top-7/SynApps/calc-2-8
-PCRE=~/top-7/pcre-7-2
+ifneq ($(filter linux%,$(T_A)),)
+PCRE_INCLUDE=/usr/include
+PCRE_LIB=/usr/lib
+endif
+
+ASYN=$(EPICS_BASE)
+CALC=$(EPICS_BASE)
+SSCAN=$(EPICS_BASE)
+EPICS_BASE=/usr/lib/epics
 
 #EPICS_BASE=/usr/local/epics/base-3.16.1
 #ASYN=~/top-3.16/asyn4-30
