From: Mike Westfall <mesamike@users.noreply.github.com>
Date: Thu, 17 Nov 2016 14:24:08 -0300
Subject: Update StreamBuffer.h

---
 src/StreamBuffer.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/StreamBuffer.h b/src/StreamBuffer.h
index a016089..e4cb6a6 100644
--- a/src/StreamBuffer.h
+++ b/src/StreamBuffer.h
@@ -23,6 +23,8 @@
 #include <string.h>
 #include <stdlib.h>
 
+#include <sys/types.h>
+
 #ifndef __GNUC__
 #define __attribute__(x)
 #endif
