From: Hugo Slepicka <slepicka@slac.stanford.edu>
Date: Fri, 27 Jan 2017 12:07:19 -0800
Subject: Removing conflict at write in devmmboDirectStream.c

---
 src/devmbboDirectStream.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/devmbboDirectStream.c b/src/devmbboDirectStream.c
index eb95110..9281436 100644
--- a/src/devmbboDirectStream.c
+++ b/src/devmbboDirectStream.c
@@ -81,7 +81,7 @@ static long initRecord (dbCommon *record)
    Thus make sure the record is never left in INVALID_ALARM status.
 */
 
-static long write(dbCommon *record)
+static long write_mbbo(dbCommon *record)
 {
     long status = streamWrite(record);
     if (record->nsev == INVALID_ALARM) record->nsev = MAJOR_ALARM;
@@ -102,7 +102,7 @@ struct {
     streamInit,
     initRecord,
     streamGetIointInfo,
-    write
+    write_mbbo
 };
 
 epicsExportAddress(dset,devmbboDirectStream);
